management/src/commands/update-identity-from-user.ts

```diff
@@ -66,4 +67,41 @@ async function updateIdentityFromUser(
   });
 }
 
+async function updateIdentityFromUser(
+  managementCredentials: ManagementCredentials,
+  userId: string,
+  validationCode: string,
+  eventIds: string[],
+  identityValue: string,
+  identityToUpdateId: string,
+  maxRetry = 2,
+): Promise<void> {
+  return await updateIdentity(
+    managementCredentials,
+    userId,
+    validationCode,
+    eventIds,
+    identityValue,
+    identityToUpdateId,
+  ).catch((error) => {
+    if (error.statusCode >= 500 || error instanceof ConnectUnreachableError) {
+      if (maxRetry > 0) {
+        return updateIdentityFromUser(
```

Comment:
What about giving it a little time before each retry depending on the number of retries? 😁

Reply:
You mean, for example, 2s for the first, 4s for the second and so on ? A kind of exponential thing ?

Reply:
Yes, Although I was thinking in the ballpark of 100ms and 500ms at first but yeah, kind of an exponential 😇

Reply:
Alright, I'm gonna look at that then 🙂 (and yes, hundreds of ms sounds better than seconds 😆 )

Reply:
Here is my take on this: 7efab63c7b2e30c070bba9fe561268b04b6120e1 🙂 

But, I have not a single idea on how I can write an unit test for that. I put `console.logs` everywhere when doing it, so I could see that there was a delay, but that's all 🤷‍♀️